book/opmanual_duckiebot/atoms_17_operation_manual_duckiebot/2_0_6_camera_data.md

```diff
@@ -42,28 +40,42 @@ You have to select the right topic from the dropdown menu:
     <img style='width:12em' src="rqt_image_view.png"/>
 </figure>
 
-
 ### Troubleshooting
 
+Symptom: When I try to do `rqt_image_view`, I don't see the window on my machine.
+
+Resolution: Sometimes window does not successfully spawn on the first try. You can Ctrl+c to terminate the prcess first, and try running that process again.
+
 Symptom: I don't see any image
 
-Reolution: Check that the `duckiebot-interface` is running
+Reolution: use `rostopic hz /![hostname]/camera_node/image/compressed` and see if the image is being published. Typically, the image publish rate should be near 30hz. 
+
+See also: For more information about `rostopic`, see [](+software_reference#rostopic).
+
+Sumptom: My image topic is not being published.
+
+Resolution: Check that the `duckiebot-interface` is running
 
-Open [the Portainer interface](#docker-setup-portainer-interface) and check the running containers. You should see one called `dt18_03_roscore_duckiebot-interface_1`.
+Open [the Portainer interface](#docker-setup-portainer-interface) and check the running containers. You should see one called `duckiebot-interface`, using image `duckietown/dt-duckiebot-interface:daffy-arm32v7`.
 
 You call also determine this by running:
 
     $ docker -H ![DUCKIEBOT_NAME].local ps
 
 and look at the output to find the Duckiebot interface container and verify that it is running.
 
+If that image is not running, you should manually start that or check to see if you `init_sd_card` procedure was correct.
+
+To manually start the `duckiebot-interface`, do: 
+
+    laptop $ docker -H ![DUCKIEBOT_NAME].local run --name duckiebot-interface -v /data:/data --privileged --network=host -dit --restart unless-stopped duckietown/dt-duckiebot-interface:daffy-arm32v7
```

Comment:
a better way that is more likely to stay up to date is to ssh into your robot and go to /config/autoboot and do docker-compose up duckiebot-interface. That way if we change the settings in the docker-compose they will automatically be consistent with this instruction

Reply:
I would personally prefer using docker -H command just because that was historically provided. However, I will add a link to page where describe the process using docker-compose up to start that container.